### ISSUE TITLE
Implement a Contributor License Agreement?

### ISSUE THREAD

**Issue by rdeltour** [w3c/epubcheck#865]
EpubCheck has benefited from many contributions from various people over the years. While it might be difficult to retroactively get these past contributors to sign a CLA, at least it could be worth clarifying the situation for any future contributions.

We can set up a CLA signing process, with a tool like [CLA Assistant](https://cla-assistant.io/) which seamlessly integrates with GitHub.

**Comment by TzviyaSiegman** [comment]
If this is functioning as a part of the EPUB 3 CG, then we might already be working under the W3C [CLA](https://www.w3.org/community/about/agreements/cla/)

**Comment by rdeltour** [comment]
Same comment as for issue #864: with the MOU+SOW setup between DAISY and the SC, I'm not sure this work can still be considered as part of the CG… to be discussed at TPAC? 😄

**Comment by rdeltour** [comment]
Following the advice of the W3C Legal team, we updated the licensing and contribution notices, see bac772199e3c578443e15a64dd7cae847351acb8. As far as I understand, a more formalized CLA setup is not particularly required or recommended.

**Comment by mattgarrish** [comment]
The only thing that strikes me as a bit weird is that the IDPF copyright in LICENSE.md only goes to 2014 and then it changes to W3C. Shouldn't the switchover date be 2017, when the merged actually happened?

**Comment by rdeltour** [comment]
> Shouldn't the switchover date be 2017, when the merger actually happened?

Oh yes, you're right (my bad for committing without proper review).

I blame a bad copy-history-rewrite, as I originally put a more correct statement in 4f572d7c5849b65c062f2fa80bb3cecd5d2dd174:
```
Copyright © 2007 Adobe Systems Incorporated
Copyright © 2008 IDPF
Copyright © 2018 W3C (MIT, ERCIM, Keio, Beihang)
```

Btw:
- based on mailing lists archives I could estimate the first release under IDPF’s umbrella to 2008, although I'm not sure about the formalities here (I found no trace of when, if ever, the project was formally transferred to IDPF, more than being a community project which loosely stewarded by IDPF). Does that sound about right?
- I think date ranges are not really needed, we can use the date of the first contribution.

**Comment by rdeltour** [comment]
> Shouldn't the switchover date be 2017, when the merger actually happened?

Fixed in 0ed1ce35c94a052218dc0fe4c639beda94c8b2af.

The copyright notice now states:

```
Copyright © 2007 Adobe Systems Incorporated
Copyright © 2008 IDPF
Copyright © 2017 W3C (MIT, ERCIM, Keio, Beihang)
```

**Comment by mattgarrish** [comment]
LGTM!
